include/ebpf_nethooks.h

```diff
@@ -175,6 +176,23 @@ EBPF_HELPER(uint64_t, bpf_sock_addr_get_current_pid_tgid, (bpf_sock_addr_t * ctx
     ((bpf_sock_addr_get_current_pid_tgid_t)BPF_FUNC_sock_addr_get_current_pid_tgid)
 #endif
 
+/**
+ * @brief Set a context for consumption by a proxy application (sock_addr specific only).
```

Comment:
```suggestion
 * @brief Set a context for consumption by a user-mode application (sock_addr specific only).
```
The program cannot tell whether it's a "proxy" application or some other type of application.